lenovo/thinkpad/x13/yoga/default.nix

```
@@ -5,4 +5,6 @@
   ];
 
   services.xserver.wacom.enable = lib.mkDefault config.services.xserver.enable;
+  
+  services.hardware.bolt.enable = lib.mkDefault true;
```

Is this useful outside of gnome?

```suggestion
  services.hardware.bolt.enable = lib.mkIf config.services.xserver.desktopManager.gnome.enable (lib.mkDefault true);
```

---

Enabling the `bolt` service brings up `boltctl`, which can be used without gnome.
Therefore, to answer your question, yes: I use it inside of `sway` without any gnome environment.